Task/Pages/search_results_page.py

```diff
@@ -0,0 +1,12 @@
+from playwright.sync_api import Page
+
+class SearchResultsPage:
+    def __init__(self, page: Page):
```

Comment:
could we make a page to be singleton accessible somewhere in the base page, so that we didn't have to init every page with it?